pubsub/topic.go

```diff
@@ -509,6 +600,25 @@ func protoToIngestionDataSourceSettings(pbs *pb.IngestionDataSourceSettings) *In
 			AWSRoleARN:        k.GetAwsRoleArn(),
 			GCPServiceAccount: k.GetGcpServiceAccount(),
 		}
+	} else if cs := pbs.GetCloudStorage(); cs != nil {
+		var format IngestionDataSourceCloudStorageInputFormat
+		switch t := cs.InputFormat.(type) {
+		case *pb.IngestionDataSourceSettings_CloudStorage_TextFormat_:
+			format = &IngestionDataSourceCloudStorageTextFormat{
+				Delimiter: *t.TextFormat.Delimiter,
+			}
+		case *pb.IngestionDataSourceSettings_CloudStorage_AvroFormat_:
+			format = &IngestionDataSourceCloudStorageAvroFormat{}
+		case *pb.IngestionDataSourceSettings_CloudStorage_PubsubAvroFormat:
+			format = &IngestionDataSourceCloudStoragePubSubAvroFormat{}
+		}
+		s.Source = &IngestionDataSourceCloudStorage{
+			State:                   CloudStorageIngestionState(cs.State),
+			Bucket:                  cs.Bucket,
```

Comment:
Is there a reason for just using `cs.Bucket` (and similarly elsewhere for this PR), instead of using getters like is done for Kinesis? I'm not too familiar with using protobufs with Go, so unsure if there's any difference between the two.

Reply:
Accessing via getters is better, will switch over.